modules/aws/src/smithy4s/aws/package.scala

```diff
@@ -32,4 +33,24 @@ package object aws {
   private[aws] def utf8String[F[_]: MonadThrow](bytes: Array[Byte]): F[String] =
     MonadThrow[F].catchNonFatal(new String(bytes, "UTF-8"))
 
+  // format: off
+  def simplify[Alg[_[_, _, _, _, _]], F[_]:ApplicativeThrow](service: Service[Alg]): service.Interpreter[AwsCall[F, *, *, *, *, *]] => service.FunctorInterpreter[F] = {
+    interpreter =>
+     new PolyFunction5[service.Operation, Kind1[F]#toKind5] {
+      override def apply[A0, A1, A2, A3, A4](op: service.Operation[A0, A1, A2, A3, A4]): F[A2] = {
```

Comment:
let's call those `I, E, O, SI, SO` as usual

Reply:
yeah I should have though of that